packages/core/src/styles/mixins/_mixins.scss

```diff
@@ -22,19 +22,20 @@ $imported-modules: () !default;
   }
 }
 
-@mixin focus-within() {
-  &:focus-within,
-  &.focus-within {
-    outline: $clr_baselineRem_2px solid Highlight;
-  }
-
-  @media (-webkit-min-device-pixel-ratio: 0) {
-    &:focus-within,
-    &.focus-within {
-      outline: $clr_baselineRem_5px auto -webkit-focus-ring-color;
-    }
-  }
-}
+// this mixin is not being used atm
```

Comment:
this mixin is not being used in core anywhere. i wasn't sure if i could remove it or not.

Reply:
I think its safe to remove. We can always find it back in source control.